example/image-classification/symbols/alexnet.py

```diff
@@ -8,39 +8,39 @@
 def get_symbol(num_classes, **kwargs):
     input_data = mx.symbol.Variable(name="data")
     # stage 1
-    conv1 = mx.symbol.Convolution(
+    conv1 = mx.symbol.Convolution(name='conv1',
         data=input_data, kernel=(11, 11), stride=(4, 4), num_filter=96)
     relu1 = mx.symbol.Activation(data=conv1, act_type="relu")
+    lrn1 = mx.symbol.LRN(data=relu1, alpha=0.0001, beta=0.75, knorm=2, nsize=5)
     pool1 = mx.symbol.Pooling(
-        data=relu1, pool_type="max", kernel=(3, 3), stride=(2,2))
-    lrn1 = mx.symbol.LRN(data=pool1, alpha=0.0001, beta=0.75, knorm=1, nsize=5)
+        data=lrn1, pool_type="max", kernel=(3, 3), stride=(2,2))
     # stage 2
-    conv2 = mx.symbol.Convolution(
-        data=lrn1, kernel=(5, 5), pad=(2, 2), num_filter=256)
+    conv2 = mx.symbol.Convolution(name='conv2',
+        data=pool1, kernel=(5, 5), pad=(2, 2), num_filter=256)
     relu2 = mx.symbol.Activation(data=conv2, act_type="relu")
-    pool2 = mx.symbol.Pooling(data=relu2, kernel=(3, 3), stride=(2, 2), pool_type="max")
-    lrn2 = mx.symbol.LRN(data=pool2, alpha=0.0001, beta=0.75, knorm=1, nsize=5)
+    lrn2 = mx.symbol.LRN(data=relu2, alpha=0.0001, beta=0.75, knorm=2, nsize=5)
+    pool2 = mx.symbol.Pooling(data=lrn2, kernel=(3, 3), stride=(2, 2), pool_type="max")
```

Comment:
It seems to me the lrn and pool layers were in opposite order to what is described in the alexnet paper. Also knorm=2 in the paper.